### PR TITLE
Update guardduty-slack as use Terraform0.12

### PR DESCRIPTION
**Pull request by ponkotuy**
## Motivation
I want to use guardduty-slack with Terraform 0.12.

## Test
```
$ mv provider.tf.test provider.tf
$ terraform validate
Success! The configuration is valid.
```

### CODE REVIEW

File: guardduty_slack/provider.tf.test [hunk 1]
```diff
@@ -0,0 +1,3 @@
+provider "aws" {
+  region = "ap-northeast-1"
+}
```
Comment (saiya) [inline guardduty_slack/provider.tf.test]: Please exclude local-test only source.

**Comment by saiya** [comment]
@ponkotuy What is _required_ (_must_) change to use this module in 0.12?

Please separate _nice-to-have_ format change from this PR. I want to keep source as same as 0.11 version as possible.

**Comment by saiya** [comment]
For example, I know `"${expression}"` can be replaced with `expression` in 0.12. But intentionally avoided to minimise diff from 0.11 branch.

**Comment by ponkotuy** [comment]
Ok, I will recreate PR.

**Pull request closed without merging**